common/app/common/configuration.scala

```diff
@@ -307,6 +307,7 @@ class GuardianConfiguration extends Logging {
     lazy val path =
       if (environment.secure) configuration.getMandatoryStringProperty("static.securePath")
       else configuration.getMandatoryStringProperty("static.path")
+    lazy val externalEmbedHost = configuration.getMandatoryStringProperty("guardian.page.externalEmbedHost")
```

Comment:
I am not sure this belongs to the `static` object. Feel free to create a more meaningful wrapper object just for it